CorsixTH/Lua/objects/machines/cardio.lua

```diff
@@ -47,6 +47,16 @@ object.usage_animations = copy_north_to_south {
     },
   },
 }
+
+TheApp.animation_manager:setStaffMarker(object.usage_animations.north.in_use.Handyman,
+  0, {-10, -7, "px"}, 1, {-13, -11, "px"}, 2, {-17, -11, "px"}, 3, {-21, -13, "px"},
+  4, {-27, -14, "px"}, 5, {-31, -16, "px"}, 6, {-37, -17, "px"}
+)
+TheApp.animation_manager:setStaffMarker(object.usage_animations.north.finish_use.Handyman,
+  0, {-37, -17, "px"}, 5, {-37, -17, "px"}, 6, {-29, -15, "px"}, 7, {-25, -15, "px"},
+  8, {-20, -13, "px"}, 9, {-19, -13, "px"}, 10, {-18, -13, "px"}, 11, {-12, -10, "px"}
+)
+
```

Comment:
Can the `kf` standard apply to this one? It all looks even more magic than before without some local vars.

Reply:
It is actually quite magic data, you show a frame in AnimView, enable mood display, and click at the picture where you think the center point is.

I made it a little easier by printing the result to the standard output, so I don't need to copy it from the gui. That patch isn't yet in. I am not sure if that is acceptable.